src/QsCompiler/Transformations/CallGraph.cs

```diff
@@ -8,6 +8,7 @@
 using Microsoft.Quantum.QsCompiler.DataTypes;
 using Microsoft.Quantum.QsCompiler.SyntaxTokens;
 using Microsoft.Quantum.QsCompiler.SyntaxTree;
+using Microsoft.Quantum.QsCompiler.Transformations.Core;
```

Comment:
Regarding TypeParameterResolutions, it would be great to have a comment in the code regarding the meaning of the key Tuple, and maybe a type definition for that would also be helpful.

Reply:
I've added a typedef (in the form of a using statement) for type parameter names and a comment to explain it. Hopefully that makes it a bit clearer.

Reply:
It is much clearer, thanks!